Car-slider.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
+      * {
+        box-sizing: border-box;
+        margin: 0;
+        padding: 0;
+        outline: none;
+      }
+
+      .card {
+        display: inline-block;
+        background-color: rgb(248, 246, 246);
+        border-radius: 5px;
+        border: 1px solid rgb(202, 202, 202);
+        margin:3px;
+      }
+
+      .card__carousel {
+        width: 62%;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        border-radius: 5px;
+      }
+
+      .card__information {
+        padding: 16px 12px 20px;
+      }
+
+      .card__name {
+        font-size: 16px;
+        color: #484848;
+        font-weight: 400;
+      }
+
+      .car__price {
+        font-size: 16px;
+        margin-top: 8px;
+        font-weight: 700;
+        color: #484848;
+      }
+
+      .card__avg-price {
+        font-size: 13px;
+        margin-top: 8px;
+        color: #6f6f6f;
+      }
+
+      .card__city-price-link {
+        color: #0288d1;
+        text-decoration: none;
+        margin-top: 8px;
+      }
+
+      .card__price-text {
+        font-size: 13px;
+        font-weight: 400;
+        color: #aaa;
+        padding-top: 2px;
+        padding-left: 3px;
+      }
+
+      .previous__card__slider,
+      .next__card__slider {
+        height: 32px;
+        width: 32px;
+        background-color: #d7d7d7;
+        position: absolute;
+        z-index: 1;
+        top: 50%;
+        border-radius: 50%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+
+      .next__card__slider {
+        right: 0;
+      }
+
+      .previous__arrow {
+        transform: rotate(180deg);
+      }
+
+      .previous__arrow,
+      .next__arrow {
+        width: 16px;
+        height: 16px;
+        color: #484848;
+      }
+
+      .card-carousel__wrapper {
+        display: flex;
+        overflow-x: scroll;
+      }
+
+      .card-carousel__wrapper::-webkit-scrollbar {
+        width: 4px;
+        height: 1px;
+      }
+      .card-carousel__wrapper::-webkit-scrollbar-thumb {
+        background-color: #c5c5c5;
+        border-radius: 12px;
+        border: 1px solid #f6f6f6;
+      }
+      .card-carousel__wrapper::-webkit-scrollbar-track {
+        background: #f6f6f6;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="card__carousel">
```

Comment:
```suggestion
    <div class="card-carousel__carousel">
```
Follow the naming convetion